lib/devise/models/password_expirable.rb

```diff
@@ -11,6 +11,19 @@ module PasswordExpirable
 
       included do
         before_save :update_password_changed
+
+        scope :with_need_change_password, -> do
+          if self.expire_password_after.is_a?(Fixnum) || self.expire_password_after.is_a?(Float)
```

Comment:
The expression `self.expire_password_after.is_a?(Fixnum) || self.expire_password_after.is_a?(Float)` appears 3 times in this module, and it's not the most readable thing in the world. Is it worth extracting it into a well-named method?